R/arrange_anova.R

```diff
@@ -135,6 +135,14 @@ arrange_anova.summary.aovlist <- function(x) {
 arrange_anova.summary.Anova.mlm <- function(x, correction = "GG") {
   validate(correction, check_class = "character", check_length = 1)
 
+  # univariate.tests is NULL if the object comes from a MANOVA
+  if(is.null(x$univariate.tests)) {
+    stop(
+      "Anova.mlm objects from car::Manova are not supported, yet. "
```

Comment:
Could you add something about requesting this on GitHub? Something along the lines of the output of `no_method()` maybe.

Reply:
Sure.